composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is a helper function to the Dataflow Template Operator Tutorial.
+# It helps the user set up a BigQuery dataset and table that is needed
+# for the tutorial.
+
+import os
+import uuid
+
+from google.cloud import bigquery
+
+from . import dataflowtemplateoperator_create_dataset_and_table_helper as helper
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+
+client = bigquery.Client()
+
+dataset_UUID = str(uuid.uuid4()).split("-")[0]
+
+expected_schema = schema = [
```

Comment:
There are two local variables; `expected_schema` and `schema`, and only `expected_schema` is used.
So maybe you can remove `schema`.

Reply:
Oops! thanks for the catch.